src/mono/wasm/runtime-test.js

```diff
@@ -6,6 +7,18 @@
 //glue code to deal with the differences between chrome, ch, d8, jsc and sm.
 var is_browser = typeof window != "undefined";
 
+//define arguments for later
+var allRuntimeArguments = [];
```

Comment:
hmm doing it this way will break all the undefined checks later, maybe we should initialize it to null here and check for that later?

Reply:
Hmm, I'm not sure. However what you suggest is more readable so I am pushing a commit to address it. Thanks for pointing it out!

Reply:
Actually, on second thought it might make sense to merge all this argument logic into 1 block because currently it is all over the place similar to how I did it in the normalize io PR